### PR TITLE
Bug/task type badge UI

### PR DESCRIPTION
**Pull request by desperado1802**
image
image
image

### CODE REVIEW

**Comment by badalkhatri0924** [comment]
image
@desperado1802 Badge missing may be only in large title (mostly there will be large title only), so check with multiple length titls

**Comment by badalkhatri0924** [comment]
### Our implementation
image

### Expected
image

@desperado1802

**Comment by desperado1802** [comment]
Now the long title doesn't make the badge to disappear:
image

Fixed the child tasks ui also:

**Comment by badalkhatri0924** [comment]
### Current implementation
image


### Expected
image


@desperado1802 Plz check each and every screen in Figma and compare with out screens, wherever require do the changes.

File: apps/web/components/pages/task/details-section/blocks/task-estimations-info.tsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ const TaskEstimationsInfo = () => {
 			>
 				<Disclosure>
 					{({ open }) => (
-						<div className="flex flex-col w-full">
+						<div className="flex flex-col w-full mt-[0.1875rem]">
```
Comment (badalkhatri0924) [inline apps/web/components/pages/task/details-section/blocks/task-estimations-info.tsx:26]: image

@desperado1802 this is still not properly centered.

File: apps/web/components/pages/task/details-section/blocks/task-progress.tsx [hunk 2]
```diff
@@ -135,7 +135,7 @@ const TaskProgress = () => {
 			<TaskRow labelTitle={trans.TOTAL_GROUP_TIME}>
 				<Disclosure>
 					{({ open }) => (
-						<div className="flex flex-col w-full">
+						<div className="flex flex-col w-full mt-[0.1875rem]">
```
Comment (badalkhatri0924) [inline apps/web/components/pages/task/details-section/blocks/task-progress.tsx:138]: image

this is also not centered properly

**Review by badalkhatri0924 — CHANGES_REQUESTED** [review]

**Review by badalkhatri0924 — APPROVED** [review]

**Pull request merged**